Submitted By: DJ Lucas (dj_at_lucasit_dot_com)
Date: 2004-02-21
Initial Package Version: 2.4.2
Origin: DJ Lucas (dj_at_lucasit_dot_com)
Description: Fix build problem in ppp-2.4.2 when compiled against
  libpcap >= 0.8.1. (contents of net/bpf.h moved to pcap-bpf.h)

diff -Naur ppp-2.4.2-orig/pppd/demand.c ppp-2.4.2/pppd/demand.c
--- ppp-2.4.2-orig/pppd/demand.c	2003-03-02 23:11:45.000000000 -0600
+++ ppp-2.4.2/pppd/demand.c	2004-02-21 17:19:26.805230344 -0600
@@ -50,7 +50,6 @@
 #include <sys/socket.h>
 #ifdef PPP_FILTER
 #include <net/if.h>
-#include <net/bpf.h>
 #include <pcap.h>
 #endif
 
diff -Naur ppp-2.4.2-orig/pppd/sys-linux.c ppp-2.4.2/pppd/sys-linux.c
--- ppp-2.4.2-orig/pppd/sys-linux.c	2004-01-12 22:05:20.000000000 -0600
+++ ppp-2.4.2/pppd/sys-linux.c	2004-02-21 17:19:43.921628256 -0600
@@ -141,7 +141,7 @@
 #endif /* IPX_CHANGE */
 
 #ifdef PPP_FILTER
-#include <net/bpf.h>
+#include <pcap-bpf.h>
 #include <linux/filter.h>
 #endif /* PPP_FILTER */
 
